timeseries/tests/Timeseries_Test.py

```diff
@@ -0,0 +1,111 @@
+import unittest
```

Comment:
Are there tests for Part 2 of the Pype lab? We're at 65% test coverage right now, below the 85% target. https://coveralls.io/builds/5564606 shows the lowest tested files